man/systemd.special.xml

```diff
@@ -285,15 +286,17 @@
         <term><filename>hibernate.target</filename></term>
         <listitem>
           <para>A special target unit for hibernating the system. This
-          pulls in <filename>sleep.target</filename>.</para>
+          pulls in <filename>sleep.target</filename> and
+          <filename>wakeup.target</filename>.</para>
```

Comment:
I think `wakeup.target` should be pulled in (Wants) by `hibernate.target` and `sleep.target` and `hybrid-sleep.target`, and should be ordered After= them. The reason to place the dependencies in .target units and not the implementing .services is that *in principle* it is possible to replace those .services by some other implementation. In practice it probably doesn't happen too often, but it seems cleaner to do this this way.